src/doc/trpl/choosing-guarantees.md

```diff
@@ -0,0 +1,417 @@
+% Wrapper Types
+
+// Alternate title: "Choosing your Guarantees"
+
+A recurring theme in Rust programming is that Rust gives one control over the costs and guarantees
+of their program.
+
+There are various "wrapper type" abstractions in the Rust standard library which embody a multitude
+of tradeoffs between cost, ergonomics, and guarantees. Many let one choose between runtime and
+compile time enforcement. This section will thoroughly explain some of these abstractions.
+
+Before proceeding, it is highly recommended that one reads about [ownership][ownership] and
+[borrowing][borrowing] in Rust.
+
+[ownership]: ownership.html
+[borrowing]: references-and-borrowing.html
+
+
+
+# Basic pointer types
+
+## `Box<T>`
+
+[`Box<T>`][box] is an "owned pointer" or a "box". While it can hand out borrowed references to the
```

Comment:
I think in this context, "pointer which is owned", is more descriptive than "box". I know that "owned pointer" is no longer a thing, but the ownership is the key feature wrt this section.